app/models/subcase.js

```diff
@@ -184,15 +184,18 @@ export default ModelWithModifier.extend({
     return latestMeeting.plannedStart;
   }),
 
-  approved: computed('decisions', function () {
+  approved: computed('treatments', function () {
     return PromiseObject.create({
-      promise: this.get('decisions').then((decisions) => {
-        const approvedDecisions = decisions.map((decision) => decision.get('approved'));
-        if (approvedDecisions && approvedDecisions.length === 0) {
+      promise: this.get('treatments').then((treatments) => {
+        const approvedTreatments = treatments.map((treatment) => {
+            return treatment.get('decisionResultCode').get('uri') == 'http://kanselarij.vo.data.gift/id/concept/beslissings-resultaat-codes/56312c4b-9d2a-4735-b0b1-2ff14bb524fd';
+          }
+        );
+        if (approvedTreatments && approvedTreatments.length === 0) {
           return false;
         }
-        const foundNonApprovedDecision = approvedDecisions.includes(false);
-        if (foundNonApprovedDecision) {
+        const foundNonApprovedTreatment = approvedTreatments.includes(false);
+        if (foundNonApprovedTreatment) {
```

Comment:
Could this be re-written as:
```javascript
// Subcase approval is true when all treatments are approved (doesn't contain one that is false)
return !approvedTreatments.includes(false);
```

Reply:
I like the readability of this one better - but I change it for you!